### PR TITLE
CSI-109 fix controller imports

### PR DESCRIPTION
**Pull request by oriyarde**
fix controller imports once and for all

### CODE REVIEW

File: controller/scripts/lint.ini [hunk 1]
```diff
@@ -23,7 +23,6 @@ ignore-patterns=csi_pb2.*[.]py
 # relevant categories: E - Error, W - Warning, R - Refactor, C - Convention
 disable=
  # easy:
- wrong-import-order, ungrouped-imports, unused-import, # C, C, W - fixable with PyCharm's Optimize Imports (Ctrl+Alt+O)
```
Comment (ArbelNathan) [inline controller/scripts/lint.ini:26]: so now we will check imports in tests?

Comment (oriyarde) [reply controller/scripts/lint.ini:26]: yes!

Comment (ArbelNathan) [reply controller/scripts/lint.ini:26]: right now we are passing it?

Comment (oriyarde) [reply controller/scripts/lint.ini:26]: yes...

**Pull request merged**